internal/graph/check.go

```diff
@@ -383,7 +383,8 @@ func (cc *ConcurrentChecker) checkDirect(ctx context.Context, crc currentRequest
 	it.Close()
 
 	// Convert the subjects into batched requests.
-	toDispatch := make([]directDispatch, 0, subjectsToDispatch.Len())
+	expectedNumberOfChunks := subjectsToDispatch.ValueLen()/int(crc.maxDispatchCount) + 1
```

Comment:
This definitely needs a comment along the lines of your PR description explaining the intention of the +1

Reply:
Done